.github/workflows/format.yml

```diff
@@ -0,0 +1,45 @@
+name: Format
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  edit:
+    name: clang-format
+    if: ${{ github.event.comment.body == '@votca-bot format' }}
```

Comment:
Do we want a different magic phrase? Or should be `startWith()` here instead of exact match?

Reply:
I think this is fine. It is simple and clear.

Reply:
uih nice